### ISSUE TITLE
[Feature Request] Remove requirement to specify android path/create file

### ISSUE THREAD

**Issue by jamesmontemagno** [Azure/azure-mobile-apps-net-client#233]
Right now Android has a special setup for creating the database file, which makes it extremely hard for new developers to get running using the SDK. 

You have to do this:

```
            var path =  "syncstore2.db";

#if __ANDROID__
            path = Path.Combine(System.Environment.GetFolderPath(System.Environment.SpecialFolder.Personal), path);

            if (!File.Exists(path))
            {
                File.Create(path).Dispose();
            }
#endif
```

This is in a shared project, but if you are in a PCL then it is even worse. Developer should be able to just pass in the file name like normal and then the library should create the file automatically like it did in the past.

**Comment by williamsrz** [comment]
I strongly agree with @jamesmontemagno developers (especially the ones who are new to the platform), should be able to get things done and the SDK, should be a tool for that, decreasing complexity besides the platform specific stuff.

**Comment by adrianhall** [comment]
Fixed with the v3.0.2 release.
